.eslintrc

```diff
@@ -38,6 +38,7 @@
     },
     "ecmaVersion": 2018,
     "sourceType": "module",
+    "allowAutomaticSingleRunInference": true,
```

Comment:
I was in the middle of updating dependencies - this just makes `yarn lint` theoretically run faster.